tests/unit_tests/test_unittests.py

```diff
@@ -1,4 +1,11 @@
-from datenguidepy.query_execution import QueryExecutioner, FieldMetaDict, TypeMetaData
```

Comment:
maybe rename file in "test_query_execution" to keep structure (and "test_unittests" in folder "unit_test" is redundant)

Reply:
True. Fortunately renaming test files is usually very simple as they don't get imported most of the time. I'm up for meaningful renaming any time.